assets/js/script.js

```diff
@@ -0,0 +1,25 @@
+const mobileMenu = document.querySelector('.mobile-menu');
+
+document.querySelector('.menu-toggle').addEventListener(
+  'click', () => {
+    mobileMenu.style.display = 'block';
+    document.body.classList.toggle('blury');
+  },
+);
+
+document.getElementById('close-mobile-menu-btn').addEventListener(
+  'click', (event) => {
+    event.preventDefault();
+    mobileMenu.style.display = 'none';
+    document.body.classList.toggle('blury');
+  },
+);
+
+document.querySelectorAll('.mobile-menu-content li a').forEach(
+  (link) => {
+    link.addEventListener('click', () => {
+      mobileMenu.style.display = 'none';
+      document.body.classList.toggle('blury');
```

Comment:
## [OPTIONAL]
- [ ] To make your code DRY(don't repeat yourself) I would suggest you put these two lines  in a separate function as these lines are repeated twice.
```js
function closeMenu () {
        mobileMenu.style.display = 'none';
        document.body.classList.toggle('blury');
}

function openMenu() {
        mobileMenu.style.display = 'block';
        document.body.classList.toggle('blury');
}

document.querySelector('.menu-toggle').addEventListener('click', openMenu),
document.getElementById('close-mobile-menu-btn').addEventListener( 'click',  closeMenu)
```

Reply:
Thank you @andersonlebon for your suggestion. I just made a toggle function to handle all three scenarios. Now the code looks better and more maintainable.

Reply:
You're welcome, sir.